blocks/hdr/hdr.css

```diff
@@ -0,0 +1,48 @@
+.hdr {
+  margin: 40px 0;
+}
+.hdr .caption {
+  margin: 5px auto 0;
+  text-align: center;
+}
+
+.hdr .hdr-container {
+  position: relative;
+}
+
+.hdr.vertical .hdr-container {
+  max-width: 500px;
+  margin: 0 auto;
+}
+
+.hdr .hdr-container.multiple img:last-child {
+  display: none;
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+.hdr .hdr-container.multiple:hover img:last-child {
+  display: block;
+  z-index: 2;
+}
+
+.hdr .hdr-container.multiple:hover:after {
+  content: 'HDR';
```

Comment:
I assume this won't need to be localized?

Reply:
Good point. My assumption is no. But, I'll have to double check.